src/js/tabs/firmware_flasher.js

```diff
@@ -222,6 +230,8 @@ firmware_flasher.initialize = function (callback) {
             buildOptionsList($('select[name="telemetryProtocols"]'), data.telemetryProtocols);
             buildOptionsList($('select[name="options"]'), data.generalOptions);
             buildOptionsList($('select[name="motorProtocols"]'), data.motorProtocols);
+            buildCustomDefinesList($('select[name="customDefines"]'), customDefinesList);
+
```

Comment:
Extra line

Reply:
What do you mean? @haslinghuis it will not work without this line. Should I call it elsewhere to load the selectable options?

Reply:
234 + 235